packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -863,6 +879,158 @@ describe('serverless cluster', () => {
 
 
   });
+
+  test('can create a Serverless cluster without vpc', () => {
+    // GIVEN
+    const stack = testStack();
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+    });
+
+    // THEN
+    expect(stack).toHaveResource('AWS::RDS::DBCluster', {
+      Engine: 'aurora-postgresql',
+      DBClusterParameterGroupName: 'default.aurora-postgresql10',
+      EngineMode: 'serverless',
+      MasterUsername: {
+        'Fn::Join': [
+          '',
+          [
+            '{{resolve:secretsmanager:',
+            {
+              Ref: 'DatabaseSecret3B817195',
+            },
+            ':SecretString:username::}}',
+          ],
+        ],
```

Comment:
Let's preserve vertical space where possible:

```suggestion
        'Fn::Join': ['', [
          '{{resolve:secretsmanager:',
          { Ref: 'DatabaseSecret3B817195' },
          ':SecretString:username::}}',
        ]],
```

Same comment in the other tests.

Reply:
Done. 

Is there a reason why you don't have prettier ? Those kind of things should be handled automatically.